packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.html

```diff
@@ -1,3 +1,8 @@
 <template>
-  <a href={result.clickUri} target={target}>{displayedTitle}</a>
-</template>
\ No newline at end of file
+  <template if:true={isSalesforceLink}>
+    <a href="#" onclick={handleOpenTabSalesforceLink} aria-label="opens in a salesforce console subTab">{displayedTitle}</a>
```

Comment:
`"opens in a salesforce console subTab"` needs to be localized.
We need to create a label and a french translation for it. check [CustomLabels.labels-meta.xml](https://github.com/coveo/ui-kit/blob/master/packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml) and [fr.translation-meta.xml](https://github.com/coveo/ui-kit/blob/master/packages/quantic/force-app/main/translations/fr.translation-meta.xml)